HTTPServerProject.Tests/HTTPServerProxyTests/ResponseTests.cs

```diff
@@ -0,0 +1,224 @@
+using HTTPServerReadTests.Streams;
+using HTTPServerProxy.Response;
+using HTTPServerWriteTests.Streams;
+
+namespace HTTPServerProxyTests.Response;
+
+public class ProxyResponseTests
+{
+    [Fact]
+    public void GetTodoPOSTValidResponseTest()
+    {
+        var expectedStatusLine = "HTTP/1.1 201 Created";
+        var proxyStream = new List<string>() 
+		{
+        "HTTP/1.1 200 OK", 
+		"Content-Type: application/json", 
+		"Content-Length: 20", 
+		"", 
+		"{ 'task': 'First Item' }" 
+		};
+        var stream = new List<string>();
```

Comment:
It's a small thing - but you should put stuff that depends on each other together. So you create the stream on line 21 but inject it on 25. You the POST and todo crated on 23 and 24 then you hop over that to use them on 27 (with the writer created in the middle). 

If you put it like:

```
var proxyStream = blah;
var reader = new TestReadStreams(proxyStream);

var stream = new List<string>();
var writer = new TestWriteStreams(stream);

var httpType = "POST";
var httpPath = "todo";
var proxyResponse = new ProxyResponse....
```

Now it's a little easier to read at a glance.

Reply:
Agreed.